package.json

```diff
@@ -13,7 +18,6 @@
     "release": "aegir release",
     "release-minor": "aegir release --type minor",
     "release-major": "aegir release --type major",
-    "build": "aegir build",
```

Comment:
I guess the removal of the `build` command is intentional?

Reply:
Yes - `prepare` is run after installing and before publishing so there's no need for a separate build command.